docs/source/roadmap.md

```diff
@@ -36,6 +36,8 @@ The items in this section are the core team's priorities.
 
 ### Next major releases
 
+- Support building mobile apps using CapacitorJS;
+- Explore bringing Redis-oplog to core;
```

Comment:
Great to see this is being moved up in the Roadmap, but wouldn't MongoDB Change Streams replace much of the need for Redis?

Reply:
Makes sense, thanks for clarifying!